bundles/org.openhab.binding.hpprinter/src/main/java/org/openhab/binding/hpprinter/internal/binder/HPPrinterBinder.java

```diff
@@ -110,74 +121,80 @@ public void dynamicallyAddChannels(ThingUID thingUid) {
             }
 
             switch (data.getType()) {
-            case SINGLECOLOR:
-                if (data.hasCumulativeMarking()) {
+                case SINGLECOLOR:
+                    if (data.hasCumulativeMarking()) {
+                        channels.add(ChannelBuilder
+                                .create(new ChannelUID(thingUid, CGROUP_USAGE, CHANNEL_COLOR_MARKING), "Number")
```

Comment:
You can use [CoreItemFactory.NUMBER](https://github.com/openhab/openhab-core/blob/master/bundles/org.openhab.core/src/main/java/org/openhab/core/library/CoreItemFactory.java#L52) to set the item type:

```suggestion
                                .create(new ChannelUID(thingUid, CGROUP_USAGE, CHANNEL_COLOR_MARKING), CoreItemFactory.NUMBER)
```